[AppKit] New warning appears after the AppKit update

The following warnings are new and are due to https://github.com/xamarin/xamarin-macios/pull/7215

```
warning CS8509: The switch expression does not handle all possible inputs (it is not exhaustive).
```

--- a/src/AppKit/NSCollectionLayoutAnchor.cs
+++ b/src/AppKit/NSCollectionLayoutAnchor.cs
@@ -19,7 +19,7 @@ namespace AppKit {
 		    {
 			    NSCollectionLayoutAnchorOffsetType.Absolute   => _LayoutAnchorWithEdgesAbsoluteOffset (edges, offset),
 			    NSCollectionLayoutAnchorOffsetType.Fractional => _LayoutAnchorWithEdgesFractionalOffset (edges, offset),
-			    _                                             => throw new ArgumentException(message: "Invalid enum value", paramName: nameof(offsetType)),
+			    _                                             => throw new ArgumentException (message: "Invalid enum value", paramName: nameof (offsetType)),
 		    };
 	}
 }

--- a/src/AppKit/NSCollectionLayoutAnchor.cs
+++ b/src/AppKit/NSCollectionLayoutAnchor.cs
@@ -17,8 +17,9 @@ namespace AppKit {
 		public static NSCollectionLayoutAnchor Create (NSDirectionalRectEdge edges, NSCollectionLayoutAnchorOffsetType offsetType, CGPoint offset) =>
 		    offsetType switch
 		    {
-			    NSCollectionLayoutAnchorOffsetType.Absolute => _LayoutAnchorWithEdgesAbsoluteOffset (edges, offset),
+			    NSCollectionLayoutAnchorOffsetType.Absolute   => _LayoutAnchorWithEdgesAbsoluteOffset (edges, offset),
 			    NSCollectionLayoutAnchorOffsetType.Fractional => _LayoutAnchorWithEdgesFractionalOffset (edges, offset),
+			    _                                             => throw new ArgumentException(message: "Invalid enum value", paramName: nameof(offsetType)),
 		    };
 	}
 }

